[MAR-3239] Master agreement (#632)
AusDTO/dto-digitalmarketplace-api: app/emails/templates/seller_edit_decline.md

Hi,  
  
Your authorised representative has the declined the new Digital Marketplace Master Agreement.  
  
This means:  

* We will deactivate your account.  
* You will not be able to apply for opportunities on the Digital Marketplace.  

> * Your business profile will be removed from the seller catalogue.  

Author: updated

  
If you want to return to the Digital Marketplace in future, you will need to:  
  
* [Contact us]({{ frontend_url }}/contact-us) to sign up again. You will not be able to manually sign up through our website.  
* Submit new requests for assessment in any of your previously approved categories.  
  
Regards,  

> The Digital Marketplace team  

Reviewer: ```suggestion

The Digital Marketplace team  
```

Hi,  
  
Your authorised representative has declined the new Digital Marketplace Master Agreement.  
  
This means:  

* We have deactivated your account.  
* You will not be able to apply for opportunities on the Digital Marketplace.  

> * Your business profile will be removed from the seller catalogue.  

Reviewer: ```suggestion
* Your business profile has been removed from the seller catalogue.  
```
